src/CommandMessenger.cpp

```diff
@@ -102,6 +104,24 @@ void OnUnknownCommand()
     cmdMessenger.sendCmd(kStatus, F("n/a"));
 }
 
+// Handles requests from the desktop app to disable power saving mode
+void OnSetPowerSavingMode()
+{
+    bool enablePowerSavingMode = cmdMessenger.readBoolArg();
+
+    // If the request is to disable power saving mode then simply set the last command
+    // to now. The next time loop() is called in mobiflight.cpp this will cause
+    // power saving mode to get turned off.
+    if (!enablePowerSavingMode) {
+#ifdef DEBUG2CMDMESSENGER
+        cmdMessenger.sendCmd(kDebug, F("Disabling power saving mode via request"));
+#endif
+        setLastCommandMillis();
+    }
+
+    // There is no handling of a request to enable powersaving mode right now.
```

Comment:
Why? This method makes one assume, you could set the mode and enter or leave PowerSavingMode.
i think we should name it so that it is clear what it does, e.g. `OnPreventPowerSavingMode()`

Reply:
Since I'm reusing the existing `kSetPowerSavingMode` event I named the handler `OnSetPowerSavingMode` to stay consistent with how all the other events are named.

It doesn't support enabling power saving mode because that would be a bunch more code changes to the firmware I didn't really want to deal with. I guess I could make it work though by setting the last command time to the earliest possible time, then that would automagically turn power saving mode on next time loop() was called.

What's your preference? Add that? Or rename the method and live with it not being consistently named like the rest?

Reply:
Turns out it was really easy to support the request to go to sleep, so I added that. Now the method name makes sense :)